### PR TITLE
USWDS Standard tables

### PR DESCRIPTION
**Pull request by bshyong**
Use USWDS styles for tables

Addresses https://github.com/department-of-veterans-affairs/vets-website/issues/2692

screen shot 2016-08-25 at 11 04 19 am
screen shot 2016-08-25 at 11 03 26 am

### CODE REVIEW

**Comment by bshyong** [comment]
@crwallace this is ready for review and merge

File: Gemfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ source 'https://rubygems.org'
 #ruby-gemset=gitool
 
 # va new gem
-gem 'va_common', '~> 0.4.1'
+# gem 'va_common', '~> 0.4.1'
```
Comment (crwallace) [inline Gemfile]: can we remove this comment?

Comment (bshyong) [inline Gemfile]: will do! nice catch

**Comment by crwallace** [comment]
LGTM!

**Pull request merged**